CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+- The :list-table: directive no longer generates incorrect warnings
```

Comment:
[nit] The surrounding colons indicate that `list-table` is a role, which it isn't. Directive usage would look like this:
```
.. list-table::
```

But it's probably reasonable to omit these decorators altogether in the changelpg!

Reply:
Thanks for catching that -- Done!